foreman/foreman.spec

```diff
@@ -719,7 +719,9 @@ sed -i 's/:organizations_enabled: false/:organizations_enabled: true/' config/se
 export BUNDLER_EXT_GROUPS="default assets"
 ln -s %{nodejs_sitelib} node_modules
 export NODE_ENV=production
+%{?scl:scl enable %{scl} "}
 webpack.js --bail --config config/webpack.config.js
```

Comment:
in the DEBs we're using `bundle exec rake webpack:compile`, maybe using that with scl_rake is more concise?